runhouse/servers/http/http_server.py

```diff
@@ -367,9 +369,14 @@ def put_resource(request: Request, message: Message):
     @staticmethod
     @app.post("/{module}/{method}")
     @validate_cluster_access
+    @tracer.start_as_current_span("call_module_method")
     def call_module_method(
         request: Request, module, method=None, message: dict = Body(default=None)
     ):
+        current_span = trace.get_current_span()
+        http_target = current_span.attributes.get("http.target", "")
```

Comment:
would we ever fail to load the current span here? Would cause the whole request to fail if so

Reply:
Yes, it's possible due to various reasons, for example if the tracer object is not present or isn't initialized.
The span returned can also be of a read-only type on some machines.
These are unlikely scenarios but I think it's still good to have a validation here, thanks.